src/qt/guiutil.cpp

```diff
@@ -354,7 +354,10 @@ boost::filesystem::path static GetAutostartDir()
 
 boost::filesystem::path static GetAutostartFilePath()
 {
-    return GetAutostartDir() / "bitcoin.desktop";
+    std::string chain = ChainNameFromCommandLine();
+    if (chain == CBaseChainParams::MAIN)
+        return GetAutostartDir() / "bitcoin.desktop";
+    return GetAutostartDir() / strprintf("bitcoin-%s.lnk", chain);
```

Comment:
I used this `+ "" +` syntax a lot in earlier days but I don't think this is suitable for a project like bitcoin: The `"+something+"` syntax usually produces unreadable and nasty diffs when changed.